### ISSUE TITLE
playground: undo

### ISSUE THREAD

**Issue by seb-odoo** [odoo/owl#337]
I would love to be able to "undo" on the playground code area. It's only possible while currently typing, but as soon as the typing area looses the focus, I can't seem to find a way to undo changes.

As the goal of a playground is to test various things, and potentially discarding them when they don't work quite right, I feel it is an important feature.

**Comment by ged-odoo** [comment]
yeah, ace editor integration is kind of limited. more work should be done. for this, I think that we need to activate the ace editor undo plugin or something
